lib/main/atom/tooltipManager.ts

```diff
@@ -15,7 +15,10 @@ import escape = require("escape-html")
 
 export function getFromShadowDom(element: JQuery, selector: string): JQuery {
   var el = element[0]
-  var found = (<any>el).rootElement.querySelectorAll(selector)
+  var found = (<any> el).querySelectorAll(selector)
+  if (!found) {
+    found = (<any> el).rootElement.querySelectorAll(selector)
+  }
```

Comment:
> I didn't check either, but if they remove rootElement, you'll get a type error trying to call querySelectorAll on undefined.

This is not actually what happens (at least not in 1.19) because it has only been deprecated, it will probably be removed later on.

I'll do 1, and include the compiled assets, and I'll later look on how to solve 2 to phase out jQuery